src/GrafOgTabell/Graf/grafLegend/grafLegend.tsx

```diff
@@ -1,19 +1,22 @@
-import { Legend, LegendProps } from 'recharts';
+import {Legend, LegendProps} from 'recharts';
```

Comment:
Burde bruke Prettier på disse filene 🙂

Reply:
Fant ut at 'organize imports' shortcut ødelegger for Prettier :( 
Mok nok kjøre Prettier engang til etter det da